docs/src/contributing.md

```diff
@@ -1,26 +1,5 @@
 # Contributing guidelines
 
-First of all, thanks for the interest!
+Great that you want to contribute to the development of Tulipa!
 
-We welcome all kinds of contribution, including, but not limited to code, documentation, examples, configuration, issue creating, etc.
-
-Be polite and respectful.
-
-## Bug reports and discussions
-
-If you think you found a bug, feel free to open an [issue](https://github.com/TulipaEnergy/TulipaPlots.jl/issues).
-Focused suggestions and requests can also be opened as issues.
-Before opening a pull request, start an issue or a discussion on the topic, please.
-
-## Working on an issue
-
-If you found an issue that interests you, comment on that issue what your plans are.
-If the solution to the issue is clear, you can immediately create a pull request (see below).
-Otherwise, say what your proposed solution is and wait for a discussion around it.
-
-> **Tip**
->
-> Feel free to ping us after a few days if there are no responses.
-
-If your solution involves code (or something that requires running the package locally), check the [developer documentation](developer.md).
-Otherwise, you can use the GitHub interface directly to create your pull request.
+Please read these guidelines [CONTRIBUTING.md](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```

Comment:
```suggestion
We follow the same guidelines as on our main repository, [TulipaEnergyModel](https://github.com/TulipaEnergy/TulipaEnergyModel.jl).
Please read these guidelines [CONTRIBUTING.md](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/CONTRIBUTING.md) and our [README.dev](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/blob/main/README.dev.md) to get you started.
```

Reply:
Also can you remove these files and just put this statement in the README.md?

Reply:
Thanks. I made the changes as requested.